testutils/src/main/java/com/amplifyframework/testutils/SynchronousApi.java

```diff
@@ -45,6 +46,10 @@
  * performing various operations.
  */
 public final class SynchronousApi {
+
+    private static final long EXTENDED_TIMEOUT_IN_MILLISECONDS =
+            TimeUnit.SECONDS.toMillis(10); // 5 seconds is insufficient
```

Comment:
this seems dubious. If we have >5 second round trips to AppSync, we have bigger problems. I suspect there’s something else going on here aside from the timeout being too short. Did you measure the response time of the actual network call?

Reply:
Actually, the problem existed mainly for the tests that were authenticated via IAM. I believe the most significant delay stemmed from IAM authentication rather than the AppSync itself. I can isolate the extended timeout just for just those tests if you would prefer it that way?

Reply:
Ahh okay! Hm, you could. Or at least a comment. A self-documenting approach would be to name the timeout like IAM_PLUS_APPSYNC_TIMEOUT or something